dockerfiles/nginx/proxito.conf.template

```diff
@@ -20,13 +20,22 @@ server {
         rewrite ^ http://$project--$version.org.dev.readthedocs.build/page/$path;
     }
 
+    # Proxy the "readthedocs-addons.js" to be downloaded from GitHub
+    location /_/static/javascript/readthedocs-addons.js {
+        proxy_pass https://raw.githubusercontent.com/readthedocs/addons/$NGINX_ADDONS_GITHUB_TAG/dist/readthedocs-addons.js;
+        add_header Content-Type	"text/javascript; charset=utf-8" always;
+    }
+
+    location /_/static/javascript/readthedocs-addons.js.map {
+        proxy_pass https://raw.githubusercontent.com/readthedocs/addons/$NGINX_ADDONS_GITHUB_TAG/dist/readthedocs-addons.js.map;
+        add_header Content-Type	"text/javascript; charset=utf-8" always;
+    }
```

Comment:
This allows us to use the exact same version of the CF Worker script in `-ops` than in `.org` repositories. Then, each time we deploy a new version of the addons, we can update `NGINX_ADDONS_GITHUB_TAG` to point to the newer tag 🚀

Reply:
Having a `rel` branch or similar would be useful here, but probably not worth doing in this PR>

Reply:
Makes sense. I need to think a little more about this. We are not following the same deploy pattern that with the other repositories, and we are always just doing the deploy from a tag.

I don't know exactly _why_, but I suppose it was to simplify things when I built the deployment (maybe I copied it from EthicalAds client? 🤔 ). However, it may worth to use the standardize it.

Reply:
OK. So, for now, I will keep it as is and we can discuss further if we need to adapt this code in the future.